### DISCUSSION TITLE
Usage of shields.io logos

### DISCUSSION THREAD

**Discussion by uncenter** [badges/shields#9299] [General] [answered]
I'm curious about why certain logos are used in certain places. On the README and on the [status/uptime page](https://stats.uptimerobot.com/PjXogHB5p), this logo is used:

![](https://raw.githubusercontent.com/badges/shields/master/readme-logo.svg?sanitize=true)

On the website, this one is used:

<img src="https://github.com/badges/shields/blob/master/frontend/static/img/logo.png?raw=true" height="100"/>

And the `shields` organization profile picture uses this slightly different one:

<img src="https://avatars.githubusercontent.com/u/6254238?v=4" height="100"/>

**Comment by chris48s** [comment, marked as answer]
There is not really any system to it.

There probably should be, but there isn't.

**Reply by uncenter** [reply]
Is there any interest in switching to the new website one?  It's the nicest looking of the three and is more recognizable than the others.

**Reply by chris48s** [reply]
I think there are cases where it is useful to have a "wide" logo, rather than a "square" one, or cases where one with text on it is useful.

<img src="https://github.com/badges/shields/blob/master/frontend/static/img/logo.png?raw=true" height="100"/> at the top of the README would look weird

![](https://raw.githubusercontent.com/badges/shields/master/readme-logo.svg?sanitize=true) would look bad as an "avatar"

More widely than that, I've never really put much thought into logos. The two (three?) logos that exist for shields existed before I first got involved with the project in 2017 and I don't think we've ever really reviewed them or put much thought into what we use where or if they are any good. It is not a topic I have particularly strong views on.
